src/xo-app/backup/overview/index.js

```diff
@@ -215,9 +217,9 @@ export default class Overview extends Component {
             <Icon icon='schedule' /> {_('backupSchedules')}
           </CardHeader>
           <CardBlock>
-            {schedules.length ? (
+            <NoObjects collection={schedules} message={_('noScheduledJobs')}>
```

Comment:
Why is the collection `schedules` and not `this._getScheduleCollection()` like the `SortedTable`?